rockusb/src/libusb.rs

```diff
@@ -256,6 +256,12 @@ impl Transport {
     pub fn reset_device(&mut self, opcode: ResetOpcode) -> Result<()> {
         self.handle_operation(crate::operation::reset_device(opcode))
     }
+
+    /// Get which mode the Rock device is booted.
+    pub fn boot_mode(&self) -> Result<BootMode> {
```

Comment:
it'd be great to hook this into the `list-device` action.

Reply:
I'm unsure what you are referring to here. In the examples section, I've added it to the `list_available_devices` function. Let me know if I need to move it to a better place.

Reply:
Perfect!